Source/Meadow.Foundation.Libraries_and_Frameworks/Displays.TextDisplayMenu/Driver/Displays.TextDisplayMenu.csproj

```diff
@@ -22,7 +22,7 @@
   <ItemGroup>
     <None Include=".\Readme.md" Pack="true" PackagePath="" />
     <None Include="..\..\..\icon.png" Pack="true" PackagePath="" />
-    <PackageReference Include="System.Text.Json" Version="7.0.2" />
     <ProjectReference Include="..\..\Graphics.MicroGraphics\Driver\Graphics.MicroGraphics.csproj" />
+    <ProjectReference Include="..\..\Serialization.MicroJson\Driver\MicroJson.csproj" />
```

Comment:
By M.F. naming conventions, shouldnt this be `Serialization.MicroJson.csproj`?

Reply:
updated